### PR TITLE
plasma-infra: Save artifacts always for "Cypress" workflow by cron

### PR DESCRIPTION
**Pull request by Yakutoc**
Добавлен флаг `with-artifacts` для "Сypress" workflow. 

Этот флаг позволяет делать override для шага "Save artifacts":

```yml
      - name: Save artifacts
        if: ${{ failure() || inputs.with-artifacts }}
        uses: actions/upload-artifact@v2
        with:
          name: Test Artifacts
          path: /home/runner/work/plasma/plasma/cypress
```

Это понадобилось для "Сypress" проверки по расписанию. 

В этой проверки мы хотим всегда сохранять результат прогона.     

---------

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.604.5473901997.swift)  
[color_asdk_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.604.5473901997.kt)  
[color_asdk_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.604.5473901997.ts)  
[color_asdk_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.604.5473901997.xml)  
[color_plasma_b2c_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.604.5473901997.swift)  
[color_plasma_b2c_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.604.5473901997.kt)  
[color_plasma_b2c_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.604.5473901997.ts)  
[color_plasma_b2c_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.604.5473901997.xml)  
[color_sberHealth_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.604.5473901997.swift)  
[color_sberHealth_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.604.5473901997.kt)  
[color_sberHealth_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.604.5473901997.ts)  
[color_sberHealth_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.604.5473901997.xml)  
[color_sbermarket_business_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.604.5473901997.swift)  
[color_sbermarket_business_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.604.5473901997.kt)  
[color_sbermarket_business_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.604.5473901997.ts)  
[color_sbermarket_business_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.604.5473901997.xml)  
[color_sbermarket_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.604.5473901997.swift)  
[color_sbermarket_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.604.5473901997.kt)  
[color_sbermarket_metro_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.604.5473901997.swift)  
[color_sbermarket_metro_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.604.5473901997.kt)  
[color_sbermarket_metro_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.604.5473901997.ts)  
[color_sbermarket_metro_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.604.5473901997.xml)  
[color_sbermarket_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.604.5473901997.ts)  
[color_sbermarket_selgros_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.604.5473901997.swift)  
[color_sbermarket_selgros_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.604.5473901997.kt)  
[color_sbermarket_selgros_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.604.5473901997.ts)  
[color_sbermarket_selgros_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.604.5473901997.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.604.5473901997.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.604.5473901997.kt)  
[color_sbermarket_wlbusiness_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.604.5473901997.ts)  
[color_sbermarket_wlbusiness_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.604.5473901997.xml)  
[color_sbermarket_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.604.5473901997.xml)  
[color_sberonline_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.604.5473901997.swift)  
[color_sberonline_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.604.5473901997.kt)  
[color_sberonline_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.604.5473901997.ts)  
[color_sberonline_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.604.5473901997.xml)  
[color_sberprime_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.604.5473901997.swift)  
[color_sberprime_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.604.5473901997.kt)  
[color_sberprime_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.604.5473901997.ts)  
[color_sberprime_xml--canary.604.5473901997.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.604.5473901997.xml)  
[shadow_sbermarket_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.604.5473901997.ts)  
[typo_mage_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.604.5473901997.swift)  
[typo_mage_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.604.5473901997.kt)  
[typo_mage_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.604.5473901997.ts)  
[typo_plasma_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.604.5473901997.swift)  
[typo_plasma_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.604.5473901997.kt)  
[typo_plasma_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.604.5473901997.ts)  
[typo_ruler_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.604.5473901997.swift)  
[typo_ruler_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.604.5473901997.kt)  
[typo_ruler_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.604.5473901997.ts)  
[typo_sage_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.604.5473901997.swift)  
[typo_sage_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.604.5473901997.kt)  
[typo_sage_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.604.5473901997.ts)  
[typo_sbermarket_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.604.5473901997.swift)  
[typo_sbermarket_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.604.5473901997.kt)  
[typo_sbermarket_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.604.5473901997.ts)  
[typo_soulmate_ios-swift--canary.604.5473901997.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.604.5473901997.swift)  
[typo_soulmate_kotlin--canary.604.5473901997.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.604.5473901997.kt)  
[typo_soulmate_react-native--canary.604.5473901997.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.604.5473901997.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.218.1-canary.604.5473901997.0
  npm install @salutejs/plasma-core@1.123.1-canary.604.5473901997.0
  npm install @salutejs/plasma-hope@1.218.1-canary.604.5473901997.0
  npm install @salutejs/plasma-icons@1.152.1-canary.604.5473901997.0
  npm install @salutejs/plasma-temple@1.167.1-canary.604.5473901997.0
  npm install @salutejs/plasma-ui@1.199.1-canary.604.5473901997.0
  npm install @salutejs/plasma-web@1.218.1-canary.604.5473901997.0
  npm install @salutejs/plasma-sb-utils@0.121.1-canary.604.5473901997.0
  # or 
  yarn add @salutejs/plasma-b2c@1.218.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-core@1.123.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-hope@1.218.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-icons@1.152.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-temple@1.167.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-ui@1.199.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-web@1.218.1-canary.604.5473901997.0
  yarn add @salutejs/plasma-sb-utils@0.121.1-canary.604.5473901997.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-604/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-604/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-604/b2c-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-604/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-604/b2c-storybook/

File: CODEOWNERS [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-* @Yeti-or @neretin-trike @Yakutoc
+* @Yeti-or @neretin-trike @Yakutoc @kayman233
```
Comment (Yakutoc) [inline CODEOWNERS:5]: @Yeti-or  А как сделать так что бы Иван стал автоматически добавляться в review список?

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-604/

File: .github/workflows/cypress-common.yml [hunk 2]
```diff
@@ -8,7 +8,9 @@ on:
         required: true
         type: string
       with-react-17:
-        required: false
```
Comment (Yeti-or) [inline .github/workflows/cypress-common.yml:11]: required не нужен?

File: CODEOWNERS (hunk 1 above)
Comment (Yeti-or) [reply CODEOWNERS:5]: кажется так и сделать

**Review by Yeti-or — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

File: .github/workflows/cypress-common.yml (hunk 2 above)
Comment (Yakutoc) [reply .github/workflows/cypress-common.yml:11]: Это опциональное свойство. Можно опустить его

**Pull request merged**